Azure-Samples/ms-identity-ciam-javascript-tutorial
# Updating the sample application for updating profile through MFA

## Purpose
Updating the sample app to reflect a api and client app to perform profile change

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd ms-identity-ciam-javascript-tutorial
git checkout [branch-name]
cd /1-Authentication/7-sign-in-express-mfa/App
npm install
npm start
```

```
cd ms-identity-ciam-javascript-tutorial
git checkout [branch-name]
cd /1-Authentication/7-sign-in-express-mfa/Api
npm install
npm start
```


* Test the code


open `http://localhost:3000/`
login and update profile from there and you should be asked for MFA

## What to Check
Verify that the following are valid
* The profile information gets updated
